package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "npm run config && webpack-dev-server --env.dev & npm run config:watch",
     "dev":"webpack-dev-server --env.dev",
-    "clean": "rimraf deploy",
+    "clean": "rm -rf deploy",
```

Comment:
rimraf won't work as is (could add as dependency or just put in the explicit bash command here)